### PR TITLE
Fix bad import in setup.py

### PR DESCRIPTION
**Pull request by glentakahashi**
Fixes https://github.com/SciTools/cartopy/issues/2021

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -30,9 +30,9 @@
 import subprocess
 import warnings
 from collections import defaultdict
+from distutils.util import convert_path
 from sysconfig import get_config_var
-
-from setuptools import Command, Extension, convert_path, setup
+from setuptools import Extension, setup
```
[the remark this answers is not in the document]
Comment (glentakahashi) [reply setup.py:35]: Unsure what to do here, want to keep it in the style of the original code, however the original code doesn't have a lint ignore comment. Should one be added?

**Comment by QuLogic** [comment]
I actually don't see any point in using `convert_path` at all, since what it's supposed to do is done in the line above its call.

In fact, I don't see the point in either, since Windows understands `/` in paths, but we can leave the manual line in there for now.

**Comment by greglucas** [comment]
@glentakahashi, sorry I opened another PR without realizing you'd already opened this (#2026). I don't think we want distutils here. I agree we can probably just remove the `convert_path` call altogether.

File: setup.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (QuLogic) [reply setup.py:35]: Don't worry about it; `setup.py` is excluded in `setup.cfg`, but I guess stickler doesn't check that config file.

**Comment by glentakahashi** [comment]
Ah @greglucas your version is better, so I'll close this one out and we should use yours

**Pull request closed without merging**